css/module.css

```diff
@@ -0,0 +1,199 @@
+    font-size: 0.8em;
+    padding:30px 0 0 0;
+    color:#888888;
+}
+
+.share-icons {
+    float:right;
+    margin-top:10px;
+}
+
+.share-icons li {
+    display:inline;
+}
+
+.share-icons li:nth-child(7) {
+    padding-left:10px;
+}
+
+/* ===================================== */
+
+/* ============== Main ================= */
+
+.recipe-name {
+    font-family:'Oswald', sans-serif;
+    font-weight:bold;
+    font-size:2em;
+    margin:30px 0 -20px 0px;
+}
+
+.post-time-stamp {
+    font-family:'Oswald', sans-serif;
+    text-transform:uppercase;
+    font-weight:lighter;
+    color:#b8b8b8;
+    font-size:0.75em;
+    margin:15px 0 0 0;
+    letter-spacing: 1px;
+    padding:15px 0 10px 0;
+}
+
+.print-container {
+    width:100%;
+}
+
+.print-icon {
+    display:inline;
+    float:right;
+    margin:72px 3px 0 0;
+    width:13px;
+    height:10px;
+}
+
+.print-text {
+    font-family:'Oswald', sans-serif;
+    text-transform:uppercase;
+    font-weight:lighter;
+    color:#b8b8b8;
+    font-size:0.75em;
+    display:inline;
+    float:right;
+    margin:70px 0 0 0;
+}
+
+.recipe-picture {
+    width:100%;
+    margin:20px 0 0 0;
+}
+
+.recipe-instructions {
+    font-family: 'Droid Serif', serif;
+    text-indent:1.5em;
+    margin:15px 0 20px 0;
+    font-size:1em;
+    color:#878787;
+    font-weight:normal;
+    line-height:1.5em;
+}
+
+.ruled-paper {
+    min-height:300px;
+    background-image: url(../img/list-bg.png);
+    background-repeat:no-repeat;
+    width:100%;
+    background-size: cover;
+}
+
+.ingredients {
+    font-family:'Patrick Hand', cursive;
+    font-size:1.40em;
+    font-weight:lighter;
+    color:#939393;
+    line-height: 1.50em;
+}
+
+.ingredients.left-side {
+    margin:40px 0 0px 75px;
+}
+
+.ingredients.right-side {
+    margin:40px 0 0px 0px;
+}
+
+
+/* =============== Author Section ========== */
+
+.author {
+    font-family:'Droid Serif', serif;
+}
+
+.author.profile-photo {
+    display:inline-block;
+    float:left;
+    margin:0 20px 0 0;
+}
+
+.author.name {
+    display:inline-block;
+    float:left;
+    font-size:1.25em;
+    font-weight: normal;
+    padding:0;
+}
+
+.author.tagline{
+    display:inline-block;
+    font-size:0.95em;
+    color:#595959;
+    float:left;
+    padding:0;
+    width:90%;
+    line-height: 1.50em;
+    margin:-40px 0 0 90px;
+}
+
+.btn-share {
+    font-family:'Oswald', sans-serif;
+    background-color:#bc7058;
+    text-transform: uppercase;
+    font-weight:lighter;
+    height:50px;
+    width:180px;
+    color:#fff;
+    border:none;
+    letter-spacing: 0.5px;
+    display:inline-block;
+    float:right;
+    margin:5px 0 0 0;
+    border-radius: 0;
+}
+
+/* ======================================== */
+
+/* ============== Footer ================== */
+
+
+.footer-section {
+    border-top: 1px solid #e2e2e2;
+    margin-top:65px;
+}
+
+.footer-logo {
+    margin:-20px 5px 20px 5px;
+    padding:0 10px 0 10px;
+    background-color: white;
+    z-index: 10;
```

Comment:
Unless this is being swallowed by another element you shouldn't need to alter its z-index.